tools/wavetool.py

```diff
@@ -0,0 +1,150 @@
+#! /usr/bin/python3
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2020 Intel Corporation. All rights reserved.
+
+"""
+A Tool to Generate and Manipulate Wave Files
+"""
+
+import os
+import sys
+import argparse
+import numpy as np
+import scipy.signal as signal
+import scipy.io.wavfile as wavefile
+
+def generate_sinusoids(**p):
+    """
+    Generate sine or cosine wave.
+
+    ``y(t) = A * sin(2 * pi * f * t + phi)``
+
+    ``y(t) = A * cos(2 * pi * f * t + phi)``
+
+    Parameters
+    ----------
+    dict of sinusiod parameters:
```

Comment:
Just a generic question. Can this function use different sine wave parameters for each channel (amp, frequency, phase)?

Reply:
Currently, no, but if we need this kind of sine wave, I can extend this feature.

Reply:
It's useful to have different signal in channels to check that one channel is not duplicated to all channels. If such is a possible fault for DSM the change should be done.

Reply:
Currently for the DSM case, we do binary comparison, so wave type/parameter don't matter. I will add this feature and chirp sound generation in my TODO list.